src/codeu/chat/ServerMain.java

```diff
@@ -98,6 +98,9 @@ private static void runServer(Uuid id,
     final Server server = new Server(id, secret, relay);
 
     LOG.info("Created server.");
+    
+    server.restoreServer(); //add to timeline
+    server.saveServer(); //execute every 30 seconds after call
```

Comment:
yes, i thought it would work here. where would you recommend putting it instead?